docs/source/models/gbd2017/causes/neonatal_other/index.rst

```diff
@@ -36,3 +36,38 @@ neonatal disorders.
 
 .. todo::
    Describe cause model
+
+Restrictions
+------------
+
+.. list-table:: Restrictions
+	:widths: 15 15 20
+	:header-rows: 1
+
+	* - Restriction type
+	  - Value
+	  - Notes
+	* - Male only
+	  - False
+	  -
+	* - Female only
+	  - False
+	  -
+	* - YLL only
+	  - False
+	  -
+	* - YLD only
+	  - False
+	  -
+	* - YLL age group start
+	  - Early neonatal
+	  - age_group_id = 2; [0-0.01917808 years)
```

Comment:
If we're going to put in age intervals, it might be nicer to use days as units for the early age groups to make it more readable. E.g. [0, 7 days) for early neonatal, and [28 days, 1 year) for post neonatal.

Reply:
Also, it shouldn't be intervals.  There is an implicit bit in the restrictions that the `age_group_start` is the left hand side of the bin represented by the id and the age_group_end is the right hand side of it's respective bin.

Reply:
I think @beatrixkh included the intervals so that the unfamiliar reader will know what the "Early neonatal" age group means. These are the intervals that Vivarium inputs returns - what's wrong with including them in the Notes column, @collijk ?

Reply:
Yes; @NathanielBlairStahn is correct, I included the intervals in the notes column as a definition of those age_groups. I can keep or delete---we can also discuss in coworking @collijk